Create Events Pages “hacktoberfest-2020/index-md” (#4319)
pulumi/docs: content/events/hacktoberfest-2020/index-md.md

---

> meta_desc: Join Pulumi for Hacktoberfest and help us build open source integrations, automation and libraries.

Reviewer: This is causing the build to fail b/c the linter expects this to be <160 characters

Author: addressed, now 99 characters.

>   learn more about cloud programming, infrastructure as code, and many other

Author: ```suggestion
```

>   topics.

Author: ```suggestion
```

block_external_search_index: false
---
## About Hacktoberfest

Hacktoberfest® is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge.

This year, Pulumi is participating in Hacktoberfest with three areas where the community at large can collaborate to build new open source capabilities:

1. **Native Pulumi Providers** - Want Pulumi to work better with your favorite cloud services? We'll show you how to get started building providers that let Pulumi users take full advantage of these services.
2. **Operator and Automation API** - Learn how to use Pulumi's new Kubernetes Superpowers to build truly automated deployments for your workloads.
3. **Reusable Infrastructure Libraries** - Create new abstractions and infrastructure building blocks using Pulumi.

## How to get started

1. **[Register](https://hacktoberfest.digitalocean.com)** on the Hacktoberfest site for access to get credit for your open source contributions.

> 2. [**Join the Pulumi Community Slack** ](https://slack.pulumi.com)and head over to the #Hacktoberfest channel.

Reviewer: This is failing the build b/c of the space b/t `* ]`

Author: fixed

> 2. [**Join the Pulumi Community Slack** ](https://slack.pulumi.com)and head over to the #Hacktoberfest channel.

Author: ```suggestion
2. [**Join the Pulumi Community Slack**](https://slack.pulumi.com)and head over to the #Hacktoberfest channel.
```

---

> meta_desc: Join Pulumi at our Infrastructure As Code Workshop at KubeCon EU and

Author: ```suggestion
meta_desc: Join Pulumi for Hacktoberfest and help us build open source integrations, automation and libraries.
```

  learn more about cloud programming, infrastructure as code, and many other

>   topics.

Reviewer: Needs updating

title: Hacktoberfest 2020
subtitle: Help us build new open source capabilities for Pulumi
unlisted: false

> redirect_to: https://hacktoberfest.digitalocean.com

Author: ```suggestion
```

event:

>   eventbrite_url: " "

Author: ```suggestion
```

  registration_url: https://hacktoberfest.digitalocean.com

>   calendly_url: " "

Author: ```suggestion
```

>   eventbrite_id: "2"

Author: ```suggestion
```

  start_date: 2020-10-22
  time: 9:00 AM - 11:00 AM
  cost: $0
  end_date: 2020-10-23
  location: VIRTUAL
  type:
    - Hackathon
  description: Join the Pulumi team and members of the community as we hack new
    integrations, automation, and infrastructure libraries!
type: events

> url_slug: workshop-amsterdam-eu-kubecon-2020

Reviewer: Needs updating

Author: fixed

> url_slug: workshop-amsterdam-eu-kubecon-2020

Author: ```suggestion
url_slug: hacktoberfest-2020
```

block_external_search_index: false
---
## About Hacktoberfest

Hacktoberfest® is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge.

This year, Pulumi is participating in Hacktoberfest with three areas where the community at large can collaborate to build new open source capabilities:

1. **Native Pulumi Providers** - Want Pulumi to work better with your favorite cloud services? We'll show you how to get started building providers that let Pulumi users take full advantage of these services.
2. **Operator and Automation API** - Learn how to use Pulumi's new Kubernetes Superpowers to build truly automated deployments for your workloads.
3. **Reusable Infrastructure Libraries** - Create new abstractions and infrastructure building blocks using Pulumi.

## How to get started

> 1. **[Register](https://hacktoberfest.digitalocean.com)** on the Hacktoberfest site for access to get credit for your open source contributions.

Author: @leezen I'll update this link after we publish on Hacktoberfest - it's a chicken and egg problem (both pages need a link referring to each other).
